velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -23,6 +23,21 @@
 
 using namespace facebook::velox::exec::test;
 using facebook::velox::VectorFuzzer;
+
+template <>
+struct fmt::formatter<facebook::velox::Date> {
+  template <typename ParseContext>
+  constexpr auto parse(ParseContext& ctx) {
+    return ctx.begin();
```

Comment:
What does this do ? Is no special logic needed here ?

Reply:
No special logic is needed since just the string formatter is used here. I followed the approach from: https://wgml.pl/blog/formatting-user-defined-types-fmt.html.